jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/DatasourceDataFetcher.java

```diff
@@ -51,6 +51,9 @@ public DatasourceData fetchDatasourceData(String datasource) {
             List<String> assetGroups = getVisibleAssetGroupsFiltered(aliases);
             datasourceData.setAssetGroups(assetGroups);
             datasourceData.setAccountIds(getAccountsByDatasource(datasource));
+            LOGGER.info("Successfully collected asset groups and account IDs");
+            LOGGER.info("Count of asset groups : {} and count of account IDs : {}", assetGroups.size(),
+                    datasourceData.getAccountIds().size());
```

Comment:
I would suggest to have it as a single message - easier to read logs when they are short.
Like "SUCCESS: Count of asset groups : {} and count of account IDs : {}"

Reply:
updated